pom.xml

```diff
@@ -66,10 +66,10 @@
     <version.org.reflections>0.9.11</version.org.reflections>
     <version.javax.inject>1</version.javax.inject>
 
-    <version.org.infinispan>10.0.0.Beta3</version.org.infinispan>
-    <version.org.infinispan.protostream>4.3.0.Alpha6</version.org.infinispan.protostream>
+    <version.org.infinispan>10.0.0.Beta5</version.org.infinispan>
+    <version.org.infinispan.protostream>4.3.0.Alpha9</version.org.infinispan.protostream>
```

Comment:
Quarkus 0.22.0 still refers to beta 3 https://github.com/kiegroup/kogito-cloud/tree/master/kogito-cloud-services